src/bubbleSort.js

```diff
@@ -0,0 +1,22 @@
+const bubbleSort = (inputArray) => {
+  let sorting = false
+  if( inputArray.length < 2) {
+    return inputArray
+  }
+  for(let index = 0; index < inputArray.length; index ++) {
+    let currentValue = inputArray[index]
+    let nextValue = inputArray[index+1]
+    if(nextValue < currentValue){
+      inputArray[index] = nextValue
+      inputArray[index+1] = currentValue
+      sorting = true
+    }
+  }
+
+  if(sorting) {
+    bubbleSort(inputArray)
```

Comment:
This recursive call smells kinda weird to me. 
The recursion is basically doing another loop for you can that would have just been done by two nested for loops.
Recursion is memory expensive and iteration is time expensive so the recursion here makes your bubble sort more 'expensive' than a traditional bubble-sort memory wise and it doesnt have to be.
For a anti-sorted array the runtime seems to be on the order of O(n^2) still so it didn't improve runtime and I looks kinda weird to me.
It works correctly though so yay you friend!